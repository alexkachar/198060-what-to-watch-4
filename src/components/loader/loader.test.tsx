import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Loader from './loader';

it(`Loader renders correctly`, () => {
  const tree = renderer
    .create(
        <Loader />, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
