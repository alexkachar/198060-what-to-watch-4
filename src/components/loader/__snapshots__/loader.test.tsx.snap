// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div
  className="loader page-content"
>
  <h1
    className="loader__text"
  >
    Loading...
  </h1>
  <div
    className="loader__spinner"
  />
</div>
`;
